app/src/androidTestInternal/java/com/rockthevote/grommet/ApplicationTest.java

```diff
@@ -1,13 +0,0 @@
-package com.rockthevote.grommet;
-
-import android.app.Application;
-import android.test.ApplicationTestCase;
-
-/**
- * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
- */
-public class ApplicationTest extends ApplicationTestCase<Application> {
-    public ApplicationTest() {
-        super(Application.class);
-    }
-}
```

Comment:
Nothing was referencing this and it was preventing any tests from running, so removed.